truewiki/views/edit.py

```diff
@@ -44,6 +44,9 @@ def save(user, old_page: str, new_page: str, content: str, payload) -> web.Respo
         create_new = False
         commit_message = f"modified: {old_page}"
 
+    if summary:
+        commit_message += f"\n User Summary: {summary}"
```

Comment:
Please add an additional \n at the beginning.

Also, is the whitespace at the start intended? `git commit` itself strips those I believe.

Reply:
Ah, I just copied in this: https://github.com/TrueBrain/TrueWiki/pull/105#discussion_r598309358 no worries

Reply:
Blame it on me :P :P :D I will :D